frontend/projects/valtimo/notify-nl/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "@valtimo/notify-nl",
```

Should change probably change. See: [link](https://github.com/generiekzaakafhandelcomponent/Plugins/pull/35/files#diff-5f263f3109b9ffa29fe164d58fc2e868ef1ad645e40800c6618435457b983687)
```suggestion
  "name": "@valtimo-plugins/notify-nl",
```